.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,6 +33,7 @@ automated e2e/unit tests.
 
 - [ ] I have **added tests** to cover my changes. (If you refactored existing code that was well tested you do not have to add tests)
 - [ ] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
+- [ ] If I made any Python code changes, I ran `black . && isort .` right before creating this pull request and added all formatting changes to my commit. Requires you to run pip install -r requirements-dev.txt to install the `black` and `isort` packages.
```

Comment:
Since this is in every PR description, I suggest that we remove the extra details from here. Instead, please update [developers.md](https://github.com/google/fhir-data-pipes/blob/master/doc/developers.md) with required steps. When updating there add a comment about doing the requirements change in a virtual-env.

```suggestion
- [ ] If I made any Python code changes, I ran `black . && isort .` right before creating this pull request and added all formatting changes to my commit.
```